feature/policy_forwarding/policy_vrf_selection/otg_tests/policy_based_traffic_steering_default_to_nondefault/README.md

```diff
@@ -0,0 +1,234 @@
+# PF-1.6: Policy forwarding based GUE tunnel decap to default and non-default network-instance
+
+## Summary
+This test ensures NOS is able to host multiple VRFs, perform GUE tunnel DECAP in the default VRF and also allows for gradual traffic migration from Default to Non-Default VRF using VRF selection policy.
+
+
+## Test environment setup
+
+### Topology
+Create the following connections:
+```mermaid
+graph LR; 
+subgraph ATE [ATE]
+    A1[Port1]
+    A2[Port2] 
+end
+subgraph DUT1 [DUT1]
+    B1[Port1]
+    B2[Port2]
+end
+A1 <-- IBGP(ASN100) & ISIS --> B1; 
+A2 <-- EBGP(ASN100:ASN200) --> B2;
+```
+
+### Configuration generation of DUT and ATE
+
+#### DUT Configuration
```

Comment:
Can you please specify whether the IP subinterfaces belong to the DEFAULT vrf or the non-default vrf?

Reply:
Added the point that the DUT's IP sub-interfaces belong to Default VRF